site/events.md

```diff
@@ -0,0 +1,33 @@
+---
+layout: page
```

Comment:
soon, we'll probably want to split this out into other layouts. one each for upcoming/past events.

Reply:
Fair enough.  I do want to get some of those pages restructured so that changes to the list become less burdensome